user/user.go

```diff
@@ -30,19 +31,19 @@ type updateUserRequest struct {
 
 // createUserResponse contains a newly created user to be returned to the client
 type createUserResponse struct {
-	ID   int64
+	ID   string
```

Comment:
perhaps this should be uuid type, seems a little bit messy having different types for it in different places